docs/rjmc.rst

```diff
@@ -15,16 +15,17 @@ Topology proposal engines
 
     TopologyProposal
     ProposalEngine
-    SmallMoleculeAtomMapper
-    PremappedSmallMoleculeSetProposalEngine
+    .. SmallMoleculeAtomMapper
+    .. PremappedSmallMoleculeSetProposalEngine
     SmallMoleculeSetProposalEngine
     PolymerProposalEngine
     PointMutationEngine
     PeptideLibraryEngine
-    NullProposalEngine
-    NaphthaleneProposalEngine
-    ButaneProposalEngine
-    PropaneProposalEngine
+    .. JRG: This class does not exist anymore?
+    .. ButaneProposalEngine
+    .. NaphthaleneProposalEngine
+    .. PropaneProposalEngine
+    .. NullProposalEngine
```

Comment:
this allows for documentation of classes? this should actually be for `topology_proposal.py`, no?